### PR TITLE
Add files for the Hub container registry feature

### PR DESCRIPTION
**Pull request by cbudz**

### CODE REVIEW

File: downstream/modules/automation-hub/proc-create-groups.adoc [hunk 1]
```diff
@@ -8,8 +8,7 @@ You can create and assign permissions to a group in Automation Hub that enables
 
 .Prerequisites
 
-* You have *groups* permissions and can create and manage group configuration and access in Automation Hub.  See xref:ref-permissions[Automation Hub permissions] for information on system permissions.
-
+* You have *groups* permissions and can create and manage group configuration and access in Automation Hub.
 .Procedure
```
Comment (TarikFD) [inline downstream/modules/automation-hub/proc-create-groups.adoc:12]:
```suggestion

.Procedure
```

Comment (TarikFD) [reply downstream/modules/automation-hub/proc-create-groups.adoc:12]: I think you need a space before .procedure to get the subheader right. Check the preview build to verify

File: downstream/modules/automation-hub/ref-container-permissions.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+= Container registry group permissions
+
+[role="_abstract"]
+User access provides granular controls to how users can interact with containers managed in private {HubName}. Use the list of permissions below to create groups with the right privileges for your container registries.
+
+.Labeled list
+Create new containers:: Users can create new containers.
+Change container namespace permissions:: Definition
+Change container:: Users can change information on a container.
+Change image tags:: Users can modify image tags
```
Comment (TarikFD) [inline downstream/modules/automation-hub/ref-container-permissions.adoc:13]:
```suggestion
Change image tags:: Users can modify image tags.
```

**Review by TarikFD — COMMENTED** [review]
Some very minor fixes. Everything else looks really good. I can merge it once you resolve these.

**Comment by cbudz** [comment]
hey @TarikFD suggestions implemented. Please feel free to merge.

**Pull request merged**